snapshots/cardano-1.26.1.yaml

```diff
@@ -0,0 +1,119 @@
+name: cardano-1.26.1
+
+resolver: lts-17.6
+
+packages:
+# Packages that exist in LTS, but which we use a different version of,
+# or just want to explicitly pin.
+- base16-bytestring-1.0.1.0 # 0.1.1.7 on LTS 17.6; would break cardano-base
+- bech32-1.1.0
+- bech32-th-1.0.2
+- Cabal-3.4.0.0
+- parsec-3.1.14.0
```

Comment:
Cabal and parsec are new. Think they were needed from cardano-transactions, but not 100%